### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'value' (for chicklet.value)

### ISSUE THREAD

**Issue by datakurre** [collective/collective.addthis#4]
Just found the following error from logs.

I'm not sure about the events that lead to this error, but clearly the viewlet could be more robust.

Thanks.

```
/var/buildout/eggs-directory/collective.addthis-1.3.1-py2.6.egg/collective/addthis/addthis.pt
- Line 15, Column 4
- Expression: <PythonExpr ('addthis_button_%s' % chicklet.value)>
- Names:
{'args': (),
...
'template':
<Products.Five.browser.pagetemplatefile.ViewPageTemplateFile object at
0x1b2eb290>,
'traverse_subpath': [],
'user': <SpecialUser 'Anonymous User'>,
'view': <Products.Five.viewlet.viewlet.SimpleViewletClass from
/var/buildout/eggs-directory/collective.addthis-1.3.1-py2.6.egg/collective/addthis/addthis.pt
object at 0x20125290>,
'views': <Products.Five.browser.pagetemplatefile.ViewMapper object at
0x201252d0>}
Module zope.tales.pythonexpr, line 59, in __call__
- __traceback_info__: ('addthis_button_%s' % chicklet.value)
Module <string>, line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'value'
```

**Comment by toutpt** [comment]
I have the same issue. comes when you work offline:

`> /Users/toutpt/.buildout/installed_eggs/collective.addthis-1.3.1-py2.6.egg/collective/addthis/addthis.py(45)chicklets()
-> return results
(Pdb) results
[None, None, None, None, None, None]`

**Comment by toutpt** [comment]
fixed in f2ea45fd1589240567c7f415a8bd52c863f22031
